spec.html

```diff
@@ -15841,8 +15841,9 @@ <h1>Runtime Semantics: Evaluation</h1>
           1. Let _func_ be ! GetSuperConstructor().
           1. Let _argList_ be ? ArgumentListEvaluation of |Arguments|.
           1. If IsConstructor(_func_) is *false*, throw a *TypeError* exception.
-          1. Let _result_ be ? Construct(_func_, _argList_, _newTarget_).
           1. Let _thisER_ be GetThisEnvironment().
+          1. If _func_ is *null*, return ? _thisER_.GetThisBinding().
```

Comment:
<code><var>func</var></code> cannot be `null` here, since the [IsConstructor](https://tc39.es/ecma262/#sec-isconstructor) call above would have already thrown a `TypeError`.